gamla/functional_generic.py

```diff
@@ -66,6 +103,19 @@ async def async_composed(*args, **kwargs):
 
 
 def compose_sync(*funcs):
```

Comment:
I have a question not related to documentation: why isn't this a private function? we are not using it directly, and I don't think we would want to (so we can be agnostic about async).
wdyt?

Reply:
I think this is a good point, I'll delete